addons/website/static/src/snippets/s_table_of_content/000.scss

```diff
@@ -11,6 +11,8 @@
             }
         }
         &.s_table_of_content_vertical_navbar .s_table_of_content_navbar {
+            word-wrap: break-word;
```

Comment:
This seems to be deprecated, replace it by `overflow-wrap: break-word;`

Reply:
@rdeodoo Fixed and updated commit message accordingly.